redfish-core/include/registries/openbmc_message_registry.hpp

```diff
@@ -634,6 +634,18 @@ constexpr std::array registry =
             },
             "None.",
         }},
+    MessageEntry{
+        "HardwareIsolationReason",
+        {
+            "Indicates the condition that affects the health of this resource.",
+            "The reason for the resource isolation: %1",
+            "OK",
```

Comment:
Was this generated from the json file?

The json entry shows "Critical" but it is not showing here. I see the same "Critical" string for other messages with it. I'm wondering if something is wrong with the generation script.

Reply:
I guess the Severity used by the script is "MessageSeverity" field from the json. And it ignores the "Severity" field. But for the hpp file that is created is matching  the original commit that i am cherry-picking. So, if I need to pull out something from the json file, I will do it in the next JIRA.